### PR TITLE
Modified the info about maintainers

### PR DESCRIPTION
**Pull request by e2**
@thibaudgg , @rymai - I wanted to change the maintainer CTA notice a bit.

I don't want people to get the impression the project is abandoned (which is far from true) - and by doing so guard is less likely to find active maintainers or even contributors.

(The notice gives the impression the project is dead and that there are bugs with no one to fix them).

Ideas for better message would be great (same probably goes for the Listen project).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### :warning: Guard is [looking for new maintainers](https://groups.google.com/forum/#!topic/guard-dev/2Td0QTvTIsE). Please [contact me](mailto:thibaud@thibaud.gg) if you're interested.
+### :exclamation::exclamation: Guard is [currently accepting more maintainers/contributors](https://groups.google.com/forum/#!topic/guard-dev/2Td0QTvTIsE). Please [contact me](mailto:thibaud@thibaud.gg) if you're interested in joining the team. :exclamation::exclamation:
```
Comment (thibaudgg) [inline README.md]: @e2 What about adding your email for the contact me link, I guess it would be more useful no?

**Comment by thibaudgg** [comment]
:+1:

**Comment by e2** [comment]
Good idea - updated.

**Comment by thibaudgg** [comment]
:+1:

**Comment by rymai** [comment]
Perfect @e2, thanks very much!

**Pull request merged**